src/main/resources/application.yml

```diff
@@ -11,6 +11,8 @@ spring:
     init:
       # to boot up application despite of any DB connection issues
       continue-on-error: true
+  main:
+    allow-bean-definition-overriding: true
```

Comment:
Why do we need this?

Reply:
This setting is particularly relevant in situations where you have multiple bean definitions with the same name in your application context. And in this case seems folio-spring-system-user has conflicting bean definitions. And if remove this setting we will get error: EcsTlrApiTest.ecsTlrIsCreated(boolean)[2] » IllegalState Failed to load ApplicationContext for [WebMergedContextConfiguration@6075a4ea testClass = org.folio.api.EcsTlrApiTest
, locations = [], classes = [org.folio.EcsTlrApplication], contextInitializerClasses = [org.folio.api.BaseIT.DockerPostgresDataSourceInitializer], activeProfiles = [], propertySourceLo
cations = [], propertySourceProperties = ["spring.kafka.consumer.auto-offset-reset=earliest", "org.springframework.boot.test.context.SpringBootTestContextBootstrapper=true", "server.po
rt=0"], contextCustomizers = [[ImportsContextCustomizer@5b0d480c key = [org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration, org.springframework.boot.test.autoconfi
gure.jdbc.TestDatabaseAutoConfiguration]], org.springframework.boot.test.context.filter.ExcludeFilterContextCustomizer@70eecdc2, org.springframework.boot.test.json.DuplicateJsonObjectC
ontextCustomizerFactory$DuplicateJsonObjectContextCustomizer@54ec8cc9, org.springframework.boot.test.mock.mockito.MockitoContextCustomizer@0, org.springframework.boot.test.web.client.T
estRestTemplateContextCustomizer@60d84f61, org.springframework.boot.test.web.reactive.server.WebTestClientContextCustomizer@2575f671, org.springframework.boot.test.autoconfigure.actuat
e.observability.ObservabilityContextCustomizerFactory$DisableObservabilityContextCustomizer@1f, org.springframework.boot.test.autoconfigure.properties.PropertyMappingContextCustomizer@
f7d54ff2, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverContextCustomizer@2d0566ba, org.springframework.test.context.support.DynamicPropertiesContextCustomizer@76125
3c0, org.springframework.boot.test.context.SpringBootTestAnnotation@2559bd8d], resourceBasePath = "src/main/webapp", contextLoader = org.springframework.boot.test.context.SpringBootContextLoader, parent = null]